github-profiles/KevinC-7394/README.md

```diff
@@ -0,0 +1,75 @@
+https://github.com/KevinCastellanos1
```

Comment:
This link should be at the end of the file

Reply:
You are right, I checked and noticed that this folder was duplicated in my files.

Reply:
You’re right,  this link was duplicated and was already deleted.